xml2rfc/writers/pdf.py

```diff
@@ -133,10 +133,12 @@ def get_mono_fonts(self):
         fonts = set()
         scripts = self.root.get('scripts').split(',')
         roboto_mono = "Roboto Mono"
+        noto_sans_mono = "Noto Sans Mono"
+        dejavu_sans_mono = "DejaVu Sans Mono"
```

Comment:
PDFs have been restricted to Noto and Roboto Mono fonts.
I guess this needs attention of rapt/rswg.